engine/Makefile

```diff
@@ -1726,6 +1726,7 @@ ifeq ($(FTE_TARGET),web)
 	EMCC_CFLAGS= -DFTE_TARGET_WEB
 	EMCC_LDFLAGS=$(EMCC_CFLAGS) $(JSLIBS) $(WEB_PREJS)
 #	EMCC_CFLAGS+= -s BINARYEN_TRAP_MODE='clamp'		#fix bigfloat->int rounding crashes
+	EMCC_CFLAGS+= -s USE_ZLIB=1
```

Comment:
This should not be needed if you're using `make makelibs FTE_TARGET=web` before building.

Reply:
Thanks for reminding me, humorously enough I'm currently flashing memtest86 to a USB-pin 🙃